spec/v3/spec.md

```diff
@@ -6,41 +6,40 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 Please refer to the [change log](./CHANGELOG.md) for a documentation of changes to this specification.
 
+## 1 Abstract
 
-## 1 Abstact
-
-PMTiles is a single-file archive format for tiled data. It enables low-cost, zero-maintenance map applications for "serverless" environments without having to rely on a custom tile backend or a third-party provider. This is achieved by packing all tiles of a tileset into an archive so that all tiles can be accessed easily and without much overhead via HTTP range requests. By combining all the tiles into one archive, hosting costs are kept low, as it is usually a lot cheaper to update one large file than to update thousands or even millions of small files.
+PMTiles is a single-file archive format for tiled data.
```

Comment:
Most specs have a short abstract, introduction or motivation paragraph at the beginning. It ensures the spec works as a standalone document and not only in this repository. I also don't think there are any real disadvantages to keeping this, since it's less than 100 words.

Reply:
I'm going to move that motivation paragraph to the README, some of the language is subjective - accessed easily, costs low, etc and doesn't fit in the spec.